rtd_requirements.txt

```diff
@@ -0,0 +1,4 @@
+mock
+pillow
+sphinx
+sphinx_rtd_theme
```

Comment:
Oops, this shouldn't be here. Dropping in PR ( https://github.com/dask/dask-image/pull/6 ).